*** Makefile.in.orig	Tue Feb 16 01:15:43 2010
--- Makefile.in	Tue Feb 16 01:14:30 2010
***************
*** 1165,1174 ****
--- 1165,1179 ----
  doxygen: 
  	cd src && doxygen
  
  doc: workbook doxygen
  
+ install-with-doc: doc install
+ 	mkdir -p $(DESTDIR)usr/share/doc/visionworkbench
+ 	cp docs/workbook/workbook.pdf $(DESTDIR)usr/share/doc/visionworkbench/
+ 	cp -r docs/doxygen $(DESTDIR)usr/share/doc/visionworkbench/apiReference
+ 
  push-coverage:
  	if test -n "$(LCOV_REMOTE_PATH)" ; then rsync -P -e ssh -r coverage/ $(LCOV_REMOTE_PATH); fi
  
  # Generate instantiation files for all templates
  instantiate-all:
